test-case/check-kmod-load-unload-after-playback.sh

```diff
@@ -85,7 +86,7 @@ do
             dlogi "platform doesn't support runtime pm, skip waiting" && break
         [[ $(sof-dump-status.py --dsp_status 0) == "suspended" ]] && break
         sleep 1
-        if [ $i -eq 15 ]; then
+        if [ "$i" -eq 15 ]; then
```

Comment:
Can you please add one more commit with this?

```suggestion
        if [ "$i" -ge 15 ]; then
```

Much safer in case someone fails to update both duplicates at the same time.

Reply:
Done.

Reply:
Why no sign-off in either commit?

Reply:
Oversight on my part, sorry.